ARM Templates/Enable Replication/azuredeploy.json

```diff
@@ -0,0 +1,121 @@
+        },
+        "recoveryRegion": {
+            "type": "String",
+            "metadata": {
+                "description": "The designated disaster recovery region where virtual machines would be brought up after failover."
+            }
+        },
+        "policyName": {
+            "defaultValue": "24-hour-replication-policy",
+            "type": "String",
+            "metadata": {
+                "description": "Name of the Replication policy to be used to create a new replication policy if protection containers are not mapped."
+            }
+        },
+        "sourceVmARMIds": {
+            "type": "String",
+            "metadata": {
+                "description": "Comma separated values of ARM IDs of the Source VMs."
+            }
+        },
+        "targetResourceGroupId": {
+            "type": "String",
+            "metadata": {
+                "description": "ARM ID of the resource group to be used to create virtual machine in DR region."
+            }
+        },
+        "targetVirtualNetworkId": {
+            "type": "String",
+            "metadata": {
+                "description": "ARM ID of the virtual network to be used by virtual machine in DR region."
+            }
+        },
+        "primaryStagingStorageAccount": {
+            "type": "String",
+            "metadata": {
+                "description": "ARM ID of the storage account to be used to cache replication data in the source region."
+            }
+        },
+        "recoveryReplicaDiskAccountType": {
+            "defaultValue": "Standard_LRS",
+            "allowedValues":[
+                "Standard_LRS",
+                "Premium_LRS",
+                "StandardSSD_LRS"
+            ],
+            "type": "String",
+            "metadata": {
+                "description": "Type of the Storage account to be used for Disk used for replication."
+            }
+        },
+        "recoveryTargetDiskAccountType": {
+            "defaultValue": "Standard_LRS",
+            "allowedValues":[
+                "Standard_LRS",
+                "Premium_LRS",
+                "StandardSSD_LRS"
+            ],            
+            "type": "String",
+            "metadata": {
+                "description": "Type of the Storage account to be used for Recovery Target Disk."
+            }
+        }
+    },
+    "resources": [
+        {
+            "type": "Microsoft.Resources/deploymentScripts",
+            "apiVersion": "2020-10-01",
+            "name": "EnableReplication",
+            "location": "[resourceGroup().location]",
+            "kind": "AzurePowerShell",
+            "identity": {
+                "type": "userAssigned",
+                "userAssignedIdentities": {
+                    "[parameters('identity')]": {}
+                }
+            },
+            "properties": {
+                "forceUpdateTag": "[utcNow()]",
+                "azPowerShellVersion": "3.0",
+                "timeout": "PT4H",
```

Comment:
azure powershell version 3, why....latest available is 7 naa

Reply:
Not sure, that's how the powershell template has been since the beginning.